src/utils/github.ts

```diff
@@ -94,14 +94,15 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | undefined> {
     logger.info({ path }, 'GitHub API: Get Content');
     const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
     if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
     if ('content' in response.data) {
       return Buffer.from(response.data.content, 'base64').toString();
     } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
+      logger.info({ path }, 'Unable to find the content from path');
+      return;
```

Comment:
This change doesn't seem to be explained in the PR description or commit messages.